translate/convert/test_po2oo.py

```diff
@@ -49,6 +49,8 @@ def check_roundtrip(self, oosource):
 
     def test_convertoo(self):
         """checks that the convertoo function is working"""
+        if os.name == 'nt':
+            skip("test or storage broken on Windows")
```

Comment:
Better use `skipif`. Don't worry, I will apply this change myself after me merging this PR.